mfjcs-rest/src/main/java/org/mfjcs/api/ItemMetadata.java

```diff
@@ -0,0 +1,11 @@
+package org.mfjcs.api;
+
+public interface ItemMetadata {
+
+	String getId();
```

Comment:
You mean like item-{uuid} vs attachment-{uuid}?
I don't think it's necessary. It will be clear from the context: url =
/items/foo, in solr it's a separate collection (or maybe just a 'type'
field so we can do a global search?)


On Wed, 24 Aug 2016 at 09:07 Marc Portier <notifications@github.com> wrote:

> In mfjcs-rest/src/main/java/org/mfjcs/api/ItemMetadata.java
> <https://github.com/marc-portier/mfjcs/pull/3#discussion_r76004433>:
>
> > @@ -0,0 +1,11 @@
> > +package org.mfjcs.api;
> > +
> > +public interface ItemMetadata {
> > +
> > +	String getId();
>
> regarding earlier remark on slack about namespacing id's
> should we introduce a tuple-string for id, or decide on a separator?
> or ignore it on this level, and just see it as part of the id being always
> 'universal'
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/marc-portier/mfjcs/pull/3/files/a9dbe4de4a65060068a13809d530a398c07cf6e0#r76004433>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJVvTeeCyxZJytzR4lmCt9xrivg4H8Hks5qi-2mgaJpZM4JrXRl>
> .
>